### ISSUE TITLE
Facing FetchError: request to https://login.botframework.com/v1/.well-known/openidconfiguration failed, reason: unable to get local issuer certificate

### ISSUE THREAD

**Issue by ashlytom** [microsoft/botbuilder-js#4650]
## Versions
What package version of the SDK are you using. : Teams toolkit v5.7.2024041607 (pre-release)
What nodejs version are you using : v20.12.2
What browser version are you using : Version 104.0.5112.81 (Official Build) (64-bit)
What os are you using - Windows 10

## Describe the bug
Facing this issue with the Team Application:
FetchError: request to https://login.botframework.com/v1/.well-known/openidconfiguration failed, reason: unable to get local issuer certificate
    at ClientRequest.<anonymous> (C:\Users\API_working_workfile\node_modules\node-fetch\lib\index.js:1501:11)
    at ClientRequest.emit (node:events:518:28)
    at ClientRequest.emit (node:domain:488:12)
    at TLSSocket.socketErrorListener (node:_http_client:500:9)
    at TLSSocket.emit (node:events:518:28)   
    at TLSSocket.emit (node:domain:488:12)   
    at emitErrorNT (node:internal/streams/destroy:169:8)
    at emitErrorCloseNT (node:internal/streams/destroy:128:3)
    at processTicksAndRejections (node:internal/process/task_queues:82:21) {
  type: 'system',
  errno: 'UNABLE_TO_GET_ISSUER_CERT_LOCALLY',  code: 'UNABLE_TO_GET_ISSUER_CERT_LOCALLY'  
}

## To Reproduce
1. Run the Debug in Chrome from VS Code
2. Once the application is launched in Chrome, type in any message in the application 
3. The error flashes in terminal

## Expected behavior
Once the applicaiton has been launched, the user inputs in the application should be captured and showed on the terminal.

## Screenshots
![image](https://github.com/microsoft/botbuilder-js/assets/65658907/3196d480-8814-4afd-a3c8-39aaaa18f6c6)
![image](https://github.com/microsoft/botbuilder-js/assets/65658907/c48633ba-362a-4113-abc6-32a1120c27cc)

**Comment by yang-112211** [comment]
I'm facing the same issue, have you resolved this? Thank you!

**Comment by ashlytom** [comment]
Yeah it was issue with an internal devtunnel API that was being blocked by the firewall.
